sherpa.sh

```diff
@@ -0,0 +1,40 @@
+package: SHERPA
+version: "%(tag_basename)s"
+source: https://github.com/alisw/sherpa
+tag: "alice/v2.1.1"
+build_requires:
+  - autotools
+  - GCC-Toolchain
+---
+#!/bin/bash -e
+
+rsync -a --delete --exclude '**/.git' --delete-excluded $SOURCEDIR/ ./
+
+autoreconf -ivf
+./configure --prefix=$INSTALLROOT \
+            --with-sqlite3=install
```

Comment:
Not yet. In principle for all generators, we could try and run a Rivet analysis on their output. We could even come up with something to validate the Rivet output. But this would still take me a moment for Sherpa to figure it out.